src/cli.rs

```diff
@@ -400,19 +400,19 @@ pub enum Command {
 }
 
 #[derive(ValueEnum, Debug, Serialize, Clone, Eq, PartialEq)]
-pub enum FormatType {
```

Comment:
renamed this to `RoverOutputFormatKind` so we know it relates to `RoverOutput` and because enums in rust are usually suffixed with `Kind` instead of `Type`

Reply:
oooh :TIL: